app.js

```diff
@@ -159,6 +156,37 @@ var eventsModule = function () {
         return treemap;
     }
 
+
+    function buildTable(ef) {
+        
+        GetAttributesValues(ef.templateName, function() {
+
+            // Select attribute values from passed event frame
+            var attributeValues = efDataHolder[ef.templateName].frames.find(e=>e.ef.webId === ef.webId).attributeValuesMap;
+            
+            // Create HTML for table and header
+            var tableContent = '<table class="exele-attr-table"><tr><th>Attribute</th><th>Value</th></tr>';
+
+            if (attributeValues) {
+                for(var attr of attributeValues) {
+
+                    if (typeof attr[1] === 'object') {
+                        // Handle object attribute type
+                    } else {
+                        tableContent += '<tr><td>' + attr[0] + '</td><td>' + attr[1] + '</td></tr>';
+                    }
+
+                }
+            }
+
+            // Close table and replace contents of existing table
+            tableContent += '</table>';
+            $('.exele-attr-table', symbolElement).replaceWith(tableContent);
```

Comment:
Ah, `replaceWith`! That's why the `<table>` element exists in the template HTML, as well.

Hm... this may be another good place for `$element.html(/* stuff */)` 😉 so that we don't need to reconstruct the `table` element. Again, no biggie, this would just shorten up a couple lines.